### ISSUE TITLE
[Bug]: Discard rates figure should not have units on y axis

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [r4ss/r4ss#860]
### Describe the bug

Rates are unitless, thus, there should not be a unit in parentheses on the y axis for the discard rate figures. 

The code causing this behavior is here:
https://github.com/r4ss/r4ss/blob/76f4627cac5b491f7e5a840e453d19d04d6c8b28/R/SSplotDiscard.R#L143-L148

### To Reproduce

plot sablefish or petrale sole using `r4ss::SSplotDiscard(model, subplot = 2)`

### Expected behavior

y axis should have the label "Discard fraction"

### Screenshots

![image](https://github.com/r4ss/r4ss/assets/4108564/bc30de56-d54d-40fb-a924-a3ec55990664)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/r4ss/r4ss#code-of-conduct)

**Comment by iantaylor-NOAA** [comment]
Good catch @kellijohnson-NOAA. I just fixed this in https://github.com/r4ss/r4ss/commit/9382f4e0a8b9ecaaf450cee3bbead3cee463814f and adc6b67 pushed to main.

I'm guessing this wasn't a typo but a confusing attempt to including the units since the ratio will be different if it's discards (mt) / (discard + retained (mt)) vs discards (1000's) / (discard + retained (1000's)). However, I think nobody cares and the axis label was misleading. Rather than lose that detail, I've moved it into the caption as in the example below. Let me know if this doesn't seem adequate.

![image](https://github.com/r4ss/r4ss/assets/4992918/bf6c422b-2c72-482a-a060-ab0a544d5f5a)
